Rubberduck.VBEditor.VB6/SafeComWrappers/VB/VBProject.cs

```diff
@@ -58,14 +59,30 @@ public EnvironmentMode Mode
         {
             get
             {
-                if (IsWrappingNullReference)
+                using (var vbe = VBE)
+                using (var mainWindow = vbe.MainWindow)
                 {
-                    return 0;
-                }
+                    var caption = mainWindow.Caption;
+
+                    if (caption.EndsWith("[design]"))
```

Comment:
IDK if that is different for VB6 or if I'm not remembering but I think it's usually proper cased (`[Design]`)? The comparison is case sensitive. It might be better to make the comparison case-insensitive so even if they managed to get it to go `[DeSiGn]` it'll still work? 

(please don't ask me how they'd manage to achieve that!)

Reply:
Good point, will do. Hoping it's not locale-sensitive...